src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/InteropServices/PlatformCompatibilityAnalyzerTests.GuardedCallsTests.cs

```diff
@@ -1055,7 +1055,7 @@ public static void WindowsSpecificApi()
 
             await VerifyAnalyzerAsyncCs(source,
                 VerifyCS.Diagnostic(PlatformCompatibilityAnalyzer.UnsupportedOsRule).WithLocation(16, 13)
-                .WithMessage("'Some.WindowsSpecificApi()' is supported on 'windows' 10.0 and later"));
+                .WithArguments("Some.WindowsSpecificApi()", "windows", "10.0"));
```

Comment:
Should be `SupportedOsVersionRule`?
As mentioned in previous comment, the test - for some reason that I don't know - will pass on its current state.

Reply:
Good catch/point, maybe it passed because has diagnostic for it warns for `UnsupportedOsRule` too, but yes `SupportedOsVersionRule` is the correct one